contracts/lib/SafeMathInt.sol

```diff
@@ -23,7 +23,8 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 */
 
-pragma solidity 0.7.6;
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Do we still need this file?

Reply:
Yes. Since the policy is still upgradable (and uses this) we want to preserve its storage layout.